Realm/RLMProperty.mm

```diff
@@ -39,7 +39,6 @@ - (instancetype)initWithName:(NSString *)name
         _objectClassName = objectClassName;
         _indexed = indexed;
         [self setObjcCodeFromType];
-        [self updateAccessors];
```

Comment:
We never needed (there) it in the first place. That initializer is only used for dynamic objects, and dynamic objects don't have accessor properties.